lib/gui/app/pages/main/controllers/main.js

```diff
@@ -93,4 +98,86 @@ module.exports = function (
       message: selectionState.getImagePath()
     }).catch(exceptionReporter.report)
   }
+
+  /**
+   * @summary Get drive title based on device quantity
+   * @function
+   * @public
+   *
+   * @returns {String} - drives title
+   *
+   * @example
+   * console.log(DriveSelectionController.getDrivesTitle())
+   * > 'Multiple Drives (4)'
+   */
+  this.getDrivesTitle = () => {
+    const drives = this.selection.getSelectedDrives()
+
+    // eslint-disable-next-line no-magic-numbers
+    if (drives.length === 1) {
+      return _.head(drives).description || 'Untitled Device'
```

Comment:
It's taken from https://github.com/balena-io/etcher/blob/add-potm-webview/lib/gui/app/pages/main/controllers/drive-selection.js#L41 , but sure

Reply:
Oh I guess that was to avoid hitting the no-magic-numbers linter rule.
Keep it as it is, I don't mind.
We should definitely remove this rule at some point.

Reply:
Yeah I agree, but in the meantime I can change it to
```javascript
/* eslint-disable no-magic-numbers */
    if (drives.length === 1) {
      return drives[0].description || 'Untitled Device'
    }
    /* eslint-enable no-magic-numbers */
```
Which will cover the whole code block